static/app/views/settings/organizationApiKeys/index.spec.jsx

```diff
@@ -54,14 +58,12 @@ describe('OrganizationApiKeys', function () {
     );
 
     expect(deleteMock).toHaveBeenCalledTimes(0);
-    wrapper.find('Confirm[aria-label="Remove API Key"]').simulate('click');
+    userEvent.click(screen.getByRole('link', {name: 'Remove API Key?'}));
 
-    const modal = await mountGlobalModal();
-    modal.find('button[aria-label="Confirm"]').simulate('click');
-
-    wrapper.update();
+    renderGlobalModal();
+    userEvent.click(screen.getByTestId('confirm-button'));
```

Comment:
can we maybe look for `screen.getByRole('button', {name:'Confirm'})`?

Reply:
I mentioned this somewhere else, but I think in the future I plan to go back through and kill the `confirm-button` data-test-id and replace all these with usages of `within`

Reply:
sorry, but why `within`? is there a case when we render multiple `Confirmation` buttons?

Reply:
Sometimes there are multiple buttons on the page with the same name.

Like i've seen a button for a row called like `Delete Item` and then in the confirm modal we name the confirm button, also, `Delete Item`. So when you try to click the confirm button, it picks-up BOTH buttons